startup.krun

```diff
@@ -0,0 +1,162 @@
+import os
+from krun.vm_defs import (PythonVMDef, LuaVMDef, JavaVMDef, GraalVMDef,
+    PHPVMDef, JRubyTruffleVMDef, V8VMDef, NativeCodeVMDef,
+    find_internal_jvmci_java_bin, GenericScriptingVMDef)
+from krun import EntryPoint
+
+# this file is a minimally hacky way to do startup measurements with all the
+# krun goodies, but without changing krun.
+#
+#
+# the way it works is as follows:
+#
+# before the invocation of the VM, another executable is inserted:
+#
+# sudo -u krun nice -20 taskset 0x2 env <some variables> \
+# startup_runner_runner_c jruby startup_runner.rb
```

Comment:
It took me a while to realise that there is a `startup_runner_c` and a `startup_runner_runner_c`. I wonder if we can find a better naming convention.